policyengine_canada/variables/gov/provinces/nu/tax/income/credits/single_status_credit/single_status_credit.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_canada.model_api import *
+
+
+class nu_single_status_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Nunavut single status credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.NU
+
+    def formula(person, period, parameters):
+        single_status = ~person.household("is_married", period)
+        amount = parameters(
```

Comment:
person should also have no dependents

Reply:
not sure how i can do this?
should i use person.household("is_dependent"). ?

Reply:
is_dependent is person level so you can just use ~person("is_dependent", period)

Reply:
(not being a dependant) ~= (not having dependants)

Reply:
wondering how to implement this?

Reply:
count_dependants = 0, could be one way